scripts/ci/diff.sh

```diff
@@ -143,19 +144,78 @@ build_msg () {
 }
 
 post_to_github () {
-  if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
-    GITHUB_SHA=$(cat $GITHUB_EVENT_PATH | jq -r .pull_request.head.sha)
-  fi
-
   if [ -z "$GITHUB_TOKEN" ]; then
     echo "Error: GITHUB_TOKEN is required to post comment to GitHub"
   else
-    echo "Posting comment to GitHub commit $GITHUB_SHA"
-    msg="$(build_msg true)"
+    if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
+      GITHUB_PULL_REQUEST_NUMBER=$(echo $github_event | jq -r .pull_request.number)
```

Comment:
I don't see `$github_event` in https://docs.github.com/en/actions/learn-github-actions/environment-variables, how's that set?

Reply:
The `load_github_env method` does `github_event=$(cat $GITHUB_EVENT_PATH)`